Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
+	public function do_nothing(): void {
+
+	}
+}
+
+// Should tell me anonymous class are not available in PHP 5.6
+$a = new class {}
+
```

Comment:
* :no_entry_sign: Anonymous classes are not supported in PHP 5.6 or earlier
* :no_entry_sign: Parenthesis should always be used when instantiating a new object.

<sub>hash: 36fa514a1d31a8bbfe4ec88f51f1d1e9</sub>